### DISCUSSION TITLE
tag seggregation in FOCUS file in FinOpsHub

### DISCUSSION THREAD

**Discussion by saleemmy** [microsoft/finops-toolkit#954] [Q&A] [answered]
Is there tag segration/split to create cost widgets per tag?

**Comment by flanakin** [comment, marked as answer]
In 0.6, we introduced an array of "promoted tags" if you edit the CostDetails query in the advanced editor. Add the tags you want to this array and we'll pull them out into separate "tag_*" columns. There's a lot more we need to do in this space, but it's a start. Let us know what you'd like to see beyond that. Feel free to drop any small (or big) ideas in the issue tracker so we can pick them up one by one as we ship each monthly release.
